CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_twilio v0.1.1
```

Comment:
Because of my upstream suggestion in the source PR, I would recommend we make this a minor upgrade change (v0.2.0).

Reply:
Let's also call out the reason for this (the dbt_utils.star update) in the transform CHANGELOG.

Reply:
added